solutions/gold/usaco-898.mdx

```diff
@@ -187,3 +187,54 @@ public class Sleepy {
 ```
 </JavaSection>
 </LanguageSection>
+
+### Using Indexed Set
+
+<LanguageSection>
+<CPPSection>
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+// Importing Indexed Sets
+#include <ext/pb_ds/assoc_container.hpp>
+using namespace __gnu_pbds;
+template <class T>
+using Tree =
+    tree<T, null_type, less<T>, rb_tree_tag, tree_order_statistics_node_update>;
+
+Tree<int> sorted;
+int N;
+vector<int> nums;
+
+int main() {
+	ifstream fin;
+	ofstream fout;
+	fin.open("sleepy.in");
+	fout.open("sleepy.out");
```

Comment:
```suggestion
	ifstream fin("sleepy.in");
	ofstream fout("sleepy.out");
```

Reply:
also put the `ofstream` near the end where you actually use it

Reply:
done with that